documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue

```diff
@@ -430,6 +430,10 @@ export default {
         }
       }
     },
+    openCreatedSpaceShortcut(parentFolder, space){
```

Comment:
No need to add a new function if it will be used once, could you put it inside the if/else block after the shortcut creation ?

Reply:
done!